frontend/integration-tests/tests/alertmanager.scenario.ts

```diff
@@ -24,7 +24,7 @@ const getGlobalsAndReceiverConfig = (configName: string, yamlStr: string) => {
   };
 };
 
-describe('Alertmanager: PagerDuty Receiver Form', () => {
+xdescribe('Alertmanager: PagerDuty Receiver Form', () => {
```

Comment:
Do we know if every one of these is flaking, or is it only certain tests?

Reply:
Hi @spadgett,  Not exactly sure, I've seen a few flakes in a few of these tests.  Should I just disable OLM and then we can try to fix anything else that fails?

Reply:
Hi @spadgett, removed the monitoring/alerting so this PR just deals with OLM.   I think other PRs will have to be made to address Cypress CRUD and Monitoring/Alertmanagement test flakes.